### ISSUE TITLE
Train and Test Batch images not being produced today?

### ISSUE THREAD

**Issue by craine** [ultralytics/yolov5#356]
## 🐛 Bug

I was running custom data and notebook and my batch images stopped running and was giving me errors. I figured it was something I did, but then I opened your colabs tutorial and tried to run that and I'm seeing the same thing happen. Maybe this is all me but I cannot get train* jpgs to work for any test. 

## To Reproduce (REQUIRED)

This is what I'm running after training in my custom notebooks which have been running for weeks and in tutorial that you created. 

Image(filename='./train_batch1.jpg', width=900)  # view augmented training mosaics
```

Output:
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-10-bf672b25fc38> in <module>()
----> 1 Image(filename='./train_batch1.jpg', width=900)  # view augmented training mosaics

3 frames
/usr/local/lib/python3.6/dist-packages/IPython/core/display.py in reload(self)
    629         """Reload the raw data from file or URL."""
    630         if self.filename is not None:
--> 631             with open(self.filename, self._read_flags) as f:
    632                 self.data = f.read()
    633         elif self.url is not None:

FileNotFoundError: [Errno 2] No such file or directory: './train_batch1.jpg'


## Expected behavior
Expecting to see images or files in my directory but they are not there. 

## Environment
If applicable, add screenshots to help explain your problem.

Colabs Pro

**Comment by glenn-jocher** [comment]
@craine thank you, yes we have a current bug in the notebooks due to a recent PR. The good news is things are a bit more organized now, so your train*.jpg and all associated training images, weights, results etc are grouped into their own runs/experiment folder.

TLDR, will fix notebook tutorials soon, your training images etc are all now located in runs/exp0, runs/exp1, etc.

**Comment by glenn-jocher** [comment]
TODO: update notebooks to conform to new runs/exp structure.

**Comment by glenn-jocher** [comment]
@craine colab and kaggle notebooks are now updated and correct. Will close as resolved now. Please let me know if there are any further issues.

- **Google Colab Notebook** with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
- **Kaggle Notebook** with free GPU: [https://www.kaggle.com/ultralytics/yolov5](https://www.kaggle.com/ultralytics/yolov5)
